tests/reference/asr-test_print2-64acb15.stderr

```diff
@@ -0,0 +1,29 @@
+Traceback (most recent call last):
```

Comment:
The expected output here should show the ASR semantic error. Here, you have a segmentation fault which means something is wrong with the code.